Cargo.toml

```diff
@@ -48,14 +48,15 @@ miden-tx              = { default-features = false, path = "crates/miden-tx", ve
 miden-tx-batch-prover = { default-features = false, path = "crates/miden-tx-batch-prover", version = "0.12" }
 
 # Miden dependencies
-miden-assembly   = { default-features = false, version = "0.17" }
-miden-core       = { default-features = false, version = "0.17" }
-miden-crypto     = { default-features = false, version = "0.15.6" }
-miden-processor  = { default-features = false, version = "0.17" }
-miden-prover     = { default-features = false, version = "0.17" }
-miden-stdlib     = { default-features = false, version = "0.17" }
-miden-utils-sync = { default-features = false, version = "0.17" }
-miden-verifier   = { default-features = false, version = "0.17" }
+miden-assembly     = { default-features = false, version = "0.17" }
+miden-core         = { default-features = false, version = "0.17" }
+miden-crypto       = { default-features = false, version = "0.15.6" }
+miden-mast-package = { default-features = false, version = "0.17.0" }
```

Comment:
Does this need to be a workspace dependency? If we're only using it in `miden-objects`, I'd add it just in its `Cargo.toml`.